src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String name;
+    private int variable;
```

Comment:
```suggestion
    private String message;
    private int year;
```
please be careful
we asked to rename variables in previous review